components/03-modules/slider/slider.hbs

```diff
@@ -0,0 +1,53 @@
+<div class="slider {{ class }}" {{{ attributes }}}>
+    <div class="slider__handler">
+        <div class="slider__frame">
+            <div class="slider__slides">
+                {{#each products }}
+                    <div class="slider__item">
+                        <a href="{{ this.link }}" class="slider__link">
+                            {{ render '@image' this.image }}
+
+                            <div class="slider__item-handler">
+                                <h2 class="heading heading--fifth-level slider__name">
+                                    {{ this.name }}
+                                </h2>
+
+                                <div class="slider__price">
+                                    <span class="slider__price-old">
+                                        {{ this.oldPrice }}
+                                    </span>
+
+                                    <span class="slider__price-new">
+                                        {{ this.price }}
+                                    </span>
+                                </div>
+                            </div>
+
+                            {{#if badge}}
+                                {{ render '@badge' this.badge }}
+                            {{/if}}
+                        </a>
+                    </div>
+                {{/each}}
+            </div>
+        </div>
+    </div>
+
+    <div class="slider__navigation">
+        <div class="slider__navigation-icon slider__navigation-icon--left slider__prev">
+            <svg title="{{ iconPrev }}" class="{{ iconClass }}">
```

Comment:
My IDE says that :
![image](https://user-images.githubusercontent.com/12099511/32666173-e661a284-c636-11e7-9491-2fbe3da13532.png)
Are you sure about using this attribute ?

Reply:
ok, talked with @anqaka and it's for accessibility, I've edited here variable and add normal text

Reply:
ok, replaced with `@button--icon` element